shell/lisa_shell

```diff
@@ -30,8 +30,13 @@ LISA_PYTHON=${LISA_PYTHON:-python3}
 # By default use internal libraries
 export LISA_DEVMODE=${LISA_DEVMODE:-1}
 
+# By default use Lisa's PS1 and colorscheme for the shell
+export LISA_PRESERVE_SHELL=${LISA_PRESERVE_SHELL:-0}
+
 # Setup colors
-source "$LISA_HOME/shell/lisa_colors"
+if [[ $LISA_PRESERVE_SHELL == 0 ]]; then
```

Comment:
we can source unconditionally and move the check inside lisa_colors, alongside the other 2 checks we have